### ISSUE TITLE
The Fetch API feature failed?

### ISSUE THREAD

**Issue by JianuoZhu** [waylaidwanderer/node-chatgpt-api#177]
hello, i setup the settings.js as the instruction, and typed my openai API. However, when I try to chat, i can't get a return from chatgpt. This is the return in terminal:
```
TypeError: fetch failed
    at Object.processResponse (node:internal/deps/undici/undici:6356:34)
    at node:internal/deps/undici/undici:6677:42
    at node:internal/process/task_queues:140:7
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at AsyncResource.runMicrotask (node:internal/process/task_queues:137:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  cause: ConnectTimeoutError: Connect Timeout Error
      at onConnectTimeout (/home/jianuo/GPT/node-chatgpt-api/node_modules/undici/lib/core/connect.js:182:24)
      at /home/jianuo/GPT/node-chatgpt-api/node_modules/undici/lib/core/connect.js:129:46
      at Immediate._onImmediate (/home/jianuo/GPT/node-chatgpt-api/node_modules/undici/lib/core/connect.js:170:9)
      at process.processImmediate (node:internal/timers:471:21) {
    code: 'UND_ERR_CONNECT_TIMEOUT'
  }
}
```
I'm sure the API key is correct, and it works well in other apps. The network is OK too. So I guess maybe Fetch API feature failed?

**Comment by waylaidwanderer** [comment]
Hey @JianuoZhu, this is a duplicate of #144.

TL;DR: likely due to China's firewall. Use a proxy or VPN.
